helm_deploy/prison-offender-events/templates/housekeeping-cronjob.yaml

```diff
@@ -20,5 +20,5 @@ spec:
               args:
                 - /bin/sh
                 - -c
-                - curl -XPUT http://prison-offender-events/queue-admin/retry-all-dlqs
```

Comment:
This is fine as is: 
![image](https://user-images.githubusercontent.com/58170926/124491475-3f88ad00-ddab-11eb-9aea-66505de75ccf.png)